site/sitecontent/terms.md

```diff
@@ -0,0 +1,264 @@
+
+## Responsibility of Contributors
+
+If you maintain a project, contribute to a project, post material to the Service, post links on the website, or otherwise make (or allow any third party to make) material available by means of the Service (any such material, “Content”), You are entirely responsible for the content of, and any harm resulting from, that Content. That is the case regardless of whether the Content in question constitutes text, graphics, a computer-aided design or computer-aided manufacturing file, an audio file, computer software or a programmable logic bitstream. By making Content available, you represent and warrant that:
+
+* the downloading, copying and use of the Content will not infringe the proprietary rights, including but not limited to the copyright, patent, trademark or trade secret rights, of any third party;
+
+* if your employer has rights to intellectual property you create, you have either (i) received permission from your employer to post or make available the Content, including but not limited to any software, or (ii) secured from your employer a waiver as to all rights in or to the Content;
+
+* you have fully complied with any third-party licences relating to the Content, and have done all things necessary to successfully pass through to end users any required terms;
+
+* the Content does not contain or install any viruses, worms, malware, Trojan horses or other harmful or destructive content;
+
+* the Content is not spam, is not machine- or randomly-generated, and does not contain unethical or unwanted commercial content designed to drive traffic to third party sites or boost the search engine rankings of third party sites, or to further unlawful acts (such as phishing) or mislead recipients as to the source of the material (such as spoofing);
+
+* the Content is not pornographic, does not contain threats or incite violence towards individuals or entities, and does not violate the privacy or publicity rights of any third party;
+
+* your project is not getting advertised via unwanted electronic messages such as spam links on newsgroups, email lists, other blogs and web sites, and similar unsolicited promotional methods;
+
+* your project is not named in a manner that misleads other users of the Service into thinking that you are another person, organisation or company;
+
+* you have, in the case of Content that includes computer software and programmable logic code, accurately categorized and/or described the type, nature, uses and effects of the materials, whether requested to do so by FOSSi Foundation or otherwise.
+
+By submitting Content to LibreCores for inclusion in a Project, you grant FOSSi Foundation a world-wide, royalty-free, and non-exclusive license to reproduce, modify, adapt and publish the Content solely for the purpose of displaying, distributing and promoting the project.
```

Comment:
This paragraph could contradict open source licenses, i.e. LibreCores wants to get broader permissions than allowed under the projects source-code license. I guess it boils down to: what's meant by "submitting Content to LibreCores for inclusion in a Project" -- is that only additional metadata people enter into LibreCores, or does it mean the project's source code? We should at least clarify that here, this paragraph is essential.